app/uk/gov/hmrc/agentmapping/connector/DesConnector.scala

```diff
@@ -53,9 +54,8 @@ class DesConnector @Inject() (@Named("des.environment") environment: String,
     }
 
   private def getRegistrationJson(utr: Utr)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[JsValue]] = {
```

Comment:
function is named "getRegistrationJson" but we are doing a post

Reply:
I think this is common use of the term "get", not related to the REST operation

Reply:
because DES is hardly REST compliant

Reply:
I guess its ok to keep the name because we are receiving something back from the request